app/src/main/java/com/kickstarter/services/gcm/RegisterService.java

```diff
@@ -35,7 +36,7 @@ public void onCreate() {
   }
 
   @Override
-  protected void onHandleIntent(final @NonNull Intent intent) {
+  protected void onHandleIntent(final @Nullable Intent intent) {
```

Comment:
the inherent overriden method has a `@Nullable` intent so made this change as per lint warning